hack/ci-kind-molecule-tests.sh

```diff
@@ -0,0 +1,441 @@
+fi
+
+if ! $CLIENT_EXE get namespace istio-system > /dev/null; then
+  if [ "${INSTALL_ISTIO}" == "true" ]; then
+    if [ ! -z "${ISTIO_VERSION}" ]; then
+      DOWNLOAD_ISTIO_VERSION_ARG="--istio-version ${ISTIO_VERSION}"
+    fi
+    hack/istio/download-istio.sh ${DOWNLOAD_ISTIO_VERSION_ARG}
+    hack/istio/install-istio-via-istioctl.sh --client-exe-path "$CLIENT_EXE"
+  else
+    infomsg "There is no 'istio-system' namespace, and this script was told not to install Istio. Aborting."
+    exit 1
+  fi
+else
+  infomsg "There is an 'istio-system' namespace - assuming Istio is installed and ready."
+fi
+
+infomsg "Building the Molecule test docker image using [${DORP}]"
+make -e FORCE_MOLECULE_BUILD="true" -e DORP="${DORP}" molecule-build
+
+mkdir -p "${LOGS_LOCAL_SUBDIR_ABS}"
+infomsg "Running the tests - logs are going here: ${LOGS_LOCAL_SUBDIR_ABS}"
+eval hack/run-molecule-tests.sh $(test ! -z "$ALL_TESTS" && echo "--all-tests \"$ALL_TESTS\"") $(test ! -z "$SKIP_TESTS" && echo "--skip-tests \"$SKIP_TESTS\"") --use-dev-images "${USE_DEV_IMAGES}" --spec-version "${SPEC_VERSION}" --helm-charts-repo "${SRC}/helm-charts" --client-exe "$CLIENT_EXE" --color false --test-logs-dir "${LOGS_LOCAL_SUBDIR_ABS}" -dorp "${DORP}" --cluster-type "kind" --operator-installer "${OPERATOR_INSTALLER:-helm}" > "${LOGS_LOCAL_RESULTS}"
+
+cd ${LOGS_LOCAL_SUBDIR_ABS}
+
+# compress large log files
+MAX_LOG_FILE_SIZE="50M"
+for bigfile in $(find ${LOGS_LOCAL_SUBDIR_ABS} -maxdepth 1 -type f -size +${MAX_LOG_FILE_SIZE})
+do
+  infomsg "This file is large and needs to be compressed: $(basename ${bigfile})"
+  tar -czf ${bigfile}.tgz -C ${LOGS_LOCAL_SUBDIR_ABS} --remove-files $(basename ${bigfile})
+done
+
+if [ "${UPLOAD_LOGS}" == "true" ]; then
+  infomsg "Committing the logs to github: ${LOGS_GITHUB_HTTPS_SUBDIR}"
+  git add -A
+  git commit -m "Test results for ${LOGS_LOCAL_SUBDIR}"
+  git push
+else
+  infomsg "The logs will not be uploaded. Test results can be found here: ${LOGS_LOCAL_SUBDIR_ABS}"
+fi
+
+# determine what message to send to IRC based on test results
+if grep FAILURE "${LOGS_LOCAL_RESULTS}"; then
+  irc_msg="a FAILURE occurred in one or more tests"
+else
+  irc_msg="all tests passed"
+fi
+
+if [ "${UPLOAD_LOGS}" == "true" ]; then
+  irc_msg="kiali tests are done [${irc_msg}]: ${LOGS_GITHUB_HTTPS_RESULTS} (test logs directory: ${LOGS_GITHUB_HTTPS_SUBDIR})"
+else
+  irc_msg="kiali tests are done [${irc_msg}]: Logs were not uploaded. See the local machine directory: ${LOGS_LOCAL_SUBDIR_ABS}"
+fi
+
+if [ "${IRC_ROOM}" == "" ]; then
+  infomsg "Not sending IRC notification - results are: ${irc_msg}"
+else
+  infomsg "Sending IRC notification to room [#${IRC_ROOM}]. msg=${irc_msg}"
+  (
+  echo 'NICK kiali-test-bot'
+  echo 'USER kiali-test-bot 8 * : kiali-test-bot'
+  sleep 10
+  echo "JOIN #${IRC_ROOM}"
+  sleep 5
+  echo "PRIVMSG #${IRC_ROOM} : ${irc_msg}"
+  echo QUIT
+  ) | nc irc.freenode.net 6667
```

Comment:
make sure to change this - going to libera now. See: https://github.com/kiali/kiali/blob/master/hack/ci-openshift-molecule-tests.sh#L441
```suggestion
  ) | nc irc.libera.chat 6667
```